queries.sql

```diff
@@ -7,4 +7,13 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered =  true;
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- AGGREGATIONS AND GROUP BY
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts  = 0;
+SELECT AVG(weight_kg) AS AVERAGE_WEIGHT FROM animals;
+SELECT neutered, count(*) FROM animals GROUP BY neutered;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, ROUND(AVG(escape_attempts), 0) AS "average number of escape attempts" FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
```

Comment:
- [ ] Great job adding your transactions to the `queries.sql` file! However, as the previous reviewer said, there are still some queries missing:

![Screen Shot 2022-08-10 at 10 18 52 AM](https://user-images.githubusercontent.com/83717285/183942738-bdec0935-51f9-4a4c-a8b0-7a5b1ec2b243.png)


It would be great to add them to your file as required by the activity.